ibis/omniscidb/tests/test_client.py

```diff
@@ -287,3 +310,24 @@ def test_cpu_execution_type(
 
     for mocked_method in mocked_methods:
         mocked_method.stop()
+
+
+@pytest.mark.parametrize(
+    'method,data',
+    [
+        ('rows', df_salary),
+        ('columnar', df_salary),
+        ('infer', df_salary),
+        ('infer', arrow_salary),
+        ('arrow', arrow_salary),
+    ],
+)
+def test_load_data(con, temp_table, method, data):
+    con.create_table(temp_table, schema=sch_salary)
+    con.load_data(temp_table, data, method=method)
```

Comment:
I would prefer if you define `df_salary`, `arrow_salary`... inside the function, and you use the strings `pandas` and `arrow` for the `data` parameter (that can be renamed to `format`). Then you can do:
```suggestion
    con.load_data(temp_table, df_salary if format == 'pandas' else arrow_salary, method=method)
```
This will save memory in the testing job and will keep things more compact.